en/docs/concepts/authentication/back-channel-logout.md

```diff
@@ -0,0 +1,22 @@
+# OpenID Connect Back-Channel Logout
+
+[OpenID Connect back-channel logout](https://openid.net/specs/openid-connect-backchannel-1_0.html#Backchannel) feature
+enables logging out users from a client application/Relying Party (RP) by directly communicating the logout requests 
+between the client application and authorization server.
+
+## What is direct communication?
+
+Direct communication enables communicating the requests between the client application and authorization server through 
+direct network links without having to rely on a browser/agent. This approach is more reliable as it does not require
+communicating the logout requests through a user agent/browser and maintain active RP browser sessions for the 
+communication to succeed.
+
+## Message flow¶
+Let's take a look at the underlying message flow of the OpenID Connect back-channel logout.
+
+1. The client application or authorization server triggers a user logout.
+2. The authorization server identifies all the client applications that share the same user session.
+3. The authorization server generates the logout token, which is a special JWT containing claims and sends it with the 
+   logout request to the logout endpoints of the client applications.
+4. Upon receiving the logout token, the client application validates the logout token and invalidates the user session.
+
```

Comment:
can we add info about back-channel communication as well? There is some content in this page: https://is.docs.wso2.com/en/latest/learn/openid-connect-single-logout/#oidc-back-channel-logout